lib/Dancer2/Cookbook.pod

```diff
@@ -120,7 +120,7 @@ to generate a graph and expects to get its data, which is in the form
 of wordcount from an AJAX call.
 
 For the graph, we need the url I</data> to return a JSON representation
-of the wordcount data. Dancer infact has a C<to_json()> function that takes
+of the wordcount data. Dancer infact has a C<encode_json()> function that takes
```

Comment:
Whoa.
Extra typo found: `infact` -> `in fact`. Extra points if you also fix that, please. :)
Also, "wordcount" should probably be "word count", no?